CONTRIBUTING.md

```diff
@@ -61,7 +61,18 @@ Pull requests
 
 When creating pull request, please heed the following:
 
-* Base your code on the latest master branch to avoid manual merges
-* Code review may ensue in order to help shape your proposal
+* Base your code on the latest `master` branch to avoid manual merges
+* Code review may ensure in order to help shape your proposal
```

Comment:
I think this introduces a typo

```suggestion
* Code review may ensue in order to help shape your proposal
```

Reply:
Can we find a wording a bit less confusing for non native english readers ?

Reply:
Sure, open for suggestions

Reply:
Are we trying to say that the user should self review or that the work will be reviewed ?

Reply:
work will be reviewed if necessary/unclear